velocidi-sample/src/main/res/layout/activity_main2.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you change the name of this file? There is no `activity_main1.xml` so having a `activity_main2.xml` is weird 😅

Reply:
Done in f90feb6